docs/integrations/app-integrations/web-crawler.mdx

```diff
@@ -25,49 +25,98 @@ CREATE DATABASE my_web
 WITH ENGINE = 'web';
 ```
 <Tip>
-The above query creates a database called `my_web`. This database by default has a table called `crawler` that we can use to crawl data from a given url/urls.
+The above query creates a database called `my_web`. This database by default has a table called `crawler` that stores data from a given URL or multiple URLs.
 </Tip>
 
 ## Usage
 
-<Note>
-Specifying a `LIMIT` clause is required. To crawl all pages on a site, consider setting the limit to a high value, such as 10,000, which exceeds the expected number of pages. Be aware that setting a higher limit may result in longer response times.
-</Note>
+### Parameters
 
-### Get Websites Content
+#### Crawl Depth
 
-The following usage examples demonstrate how to retrieve content from `docs.mindsdb.com`:
+The `crawl_depth` parameter defines how deep the crawler should navigate through linked pages:
+
+- `crawl_depth = 0`: Crawls only the specified page.
+- `crawl_depth = 1`: Crawls the specified page and all linked pages on it.
+- Higher values continue the pattern.
+
+#### Page Limits
+
+There are multiple ways to limit the number of pages returned:
+
+- `LIMIT`: Defines the maximum number of pages returned globally.
+- `per_url_limit`: Limits the number of pages returned for each specific URL, if more than one URL is provided.
```

Comment:
@martyna-mindsdb Should we maybe mention the fact that the first is the `LIMIT` clause when issuing the query and `per_url_limit` is a condition?

Reply:
Good idea. I changed it to this:

- The `LIMIT` clause defines the maximum number of pages returned globally.
- The `per_url_limit` parameter limits the number of pages returned for each specific URL, if more than one URL is provided.